### DISCUSSION TITLE
Export gefilterte Anteilsdaten per Exportknopf

### DISCUSSION THREAD

**Discussion by budda85** [OpenOlitor/OpenOlitor#123] [Q&A]
Wenn ich mir meine Anteile in OO per Filter sortiere, wäre es super, das ich mir auch nur diese Ansicht als ODS exportieren kann. Wenn ich allerdings auf den Exportknopf drücke, wird mir der komplette Datensatz exportiert.
Ist das möglich bzw. kann man das irgendwo einstellen?
Das ist bei den Mitlgiedern das selbe.

![grafik](https://user-images.githubusercontent.com/20152200/210315562-3e0f4e4d-569d-4a1c-a7e2-1c38b82acc40.png)

**Comment by AlwinEgger** [comment]
Danke für den Input. Der Wunsch ist nachvollziehbar und sinnvoll. Als Fallback kann die Filterung auch im Calc/Excel gemacht werden.

Es gibt hierzu schon länger einen internen Task https://github.com/OpenOlitor/operations/issues/92.

**Comment by budda85** [comment]
Ja Filtern tun wir das auch im Calc, aber dauert der Export natürlich schon deutlich länger wenn alles exportiert wird.
Und man würde sich einen Arbeitschritt sparen wenn es passend exportiert werden würde.

**Reply by AlwinEgger** [reply]
Absolut, das sehe ich absolut. Gegenwärtig haben wir gerade keinen Weiterentwicklungsschritt geplant, werden dies dann aber auf jeden Fall einfliessen lassen. Danke!
